src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -236,5 +186,58 @@ private static void PublishFiles(IEnumerable<LibraryAsset> files, string outputP
                 File.Copy(file.ResolvedPath, Path.Combine(outputPath, Path.GetFileName(file.ResolvedPath)), overwrite: true);
             }
         }
+
+        public static int Main(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            var app = new CommandLineApplication();
+            app.Name = "dotnet publish";
+            app.FullName = ".NET Publisher";
+            app.Description = "Publisher for the .NET Platform";
+            app.HelpOption("-h|--help");
+
+            var framework = app.Option("-f|--framework <FRAMEWORK>", "Target framework to compile for", CommandOptionType.SingleValue);
+            var runtime = app.Option("-r|--runtime <RUNTIME_IDENTIFIER>", "Target runtime to publish for", CommandOptionType.SingleValue);
+            var output = app.Option("-o|--output <OUTPUT_PATH>", "Path in which to publish the app", CommandOptionType.SingleValue);
+            var configuration = app.Option("-c|--configuration <CONFIGURATION>", "Configuration under which to build", CommandOptionType.SingleValue);
+            var projectPath = app.Argument("<PROJECT>", "The project to publish, defaults to the current directory. Can be a path to a project.json or a project directory");
+
+            app.OnExecute(() =>
+            {
+                var publish = new PublishCommand();
+
+                publish.Framework = framework.Value();
+                publish.OutputPath = output.Value();
+
```

Comment:
My publish tests would have caught this. Need to get that in soon.

https://github.com/Sridhar-MS/cli/blob/publish-tests/test/Microsoft.DotNet.Tools.Publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs#L28